src/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Today's To-Do</title>
+  </head>
+  <body>
+    <header class="head"></header>
+    <div class="crud">
+      <h2>Today's To-Do</h2>
```

Comment:
Please note that starting with the `<h1>` tag greatly improves HTML semantics. You can simply style the `<h1>` to suit the style you want.